src/main/java/eu/darkbot/api/plugin/Feature.java

```diff
@@ -0,0 +1,29 @@
+package eu.darkbot.api.plugin;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ *
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface Feature {
+
+    /**
+     * Name of the {@link Feature}
+     */
+    String name();
+
+    /**
+     * Description of the {@link Feature}
+     */
+    String description();
+
+    /**
+     * Should feature be enabled by default?
```

Comment:
no, you'll enable a feature if you want it otherwise it should stay disabled. Modules are (or rather, have been) an exception since an enabled module does nothing unless you select it